src/signals/incident-management/containers/IncidentDetail/components/Header/components/DownloadButton/index.js

```diff
@@ -18,9 +18,13 @@ function handleDownload(url, filename, accessToken) {
     responseType: 'blob'
   }).then((response) => response.blob())
     .then((blob) => {
-      const link = document.querySelector('.download-button__link');
-      link.href = window.URL.createObjectURL(blob);
-      link.click();
+      if (navigator.msSaveOrOpenBlob) {
```

Comment:
Don't forget to write a test for this.
See https://github.com/Amsterdam/registraties/blob/9e56932ad271dfb29fc574edd25476238c5e5c7f/app/containers/CSVDownload/__tests__/CSVDownload.test.js#L235 for reference.

Reply:
doen we later